lib/dataContract/DataContract.js

```diff
@@ -1,20 +1,33 @@
+const bs58 = require('bs58');
 const hash = require('../util/hash');
 const { encode } = require('../util/serializer');
 
 const InvalidDocumentTypeError = require('../errors/InvalidDocumentTypeError');
 
 class DataContract {
   /**
-   * @param {string} contractId
-   * @param {Object<string, Object>} documents
+   * @param {RawDataContract} rawDataContract
    */
-  constructor(contractId, documents) {
-    this.contractId = contractId;
+  constructor(rawDataContract) {
+    this.ownerId = rawDataContract.$ownerId;
+    this.entropy = rawDataContract.$entropy;
 
     this.setVersion(DataContract.DEFAULTS.VERSION);
     this.setJsonMetaSchema(DataContract.DEFAULTS.SCHEMA);
-    this.setDocuments(documents);
+    this.setDocuments(rawDataContract.documents);
     this.setDefinitions({});
+
+    if (rawDataContract.$schema) {
```

Comment:
If think we don't  need conditions here

Reply:
@shumkov how we can setup default values then?

Reply:
Yes, but I believe we need to pass them from the factory.